pandas/core/generic.py

```diff
@@ -10260,6 +10260,13 @@ def _logical_func(
     ):
         nv.validate_logical_func((), kwargs, fname=name)
         if level is not None:
+            warnings.warn(
+                "Using the level keyword in DataFrame and Series aggregations is "
```

Comment:
and same for all of these examples (change the example function to be applicable)

Reply:
Done